app/src/main/java/com/iron/espresso/presentation/home/study/list/StudyListActivity.kt

```diff
@@ -0,0 +1,93 @@
+package com.iron.espresso.presentation.home.study.list
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
+import com.google.android.material.tabs.TabLayout
+import com.iron.espresso.R
+import com.iron.espresso.base.BaseActivity
+import com.iron.espresso.databinding.ActivityStudyListBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class StudyListActivity :
+    BaseActivity<ActivityStudyListBinding>(R.layout.activity_study_list) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setToolbarTitle(getString(R.string.study))
+        setNavigationIcon(R.drawable.ic_back_24)
+
+        val fragmentManager = supportFragmentManager
+
+        fragmentManager.beginTransaction()
+            .replace(
+                R.id.study_list_container,
+                NewListFragment.newInstance(),
+                resources.getString(R.string.recency)
+            ).commit()
```

Comment:
```suggestion
        fragmentManager.beginTransaction()
            .replace(
                R.id.study_list_container,
                NewListFragment.newInstance(),
                resources.getString(R.string.recency)
            )
            .commit()
```
메서드 chaining convention [Changed call wrapping](https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping)

Reply:
convention에 맞게 수정하였습니다